de-AT German-Austria (Deutsch-Österreich)/Oqtane.Client/Resources/Installer/Controls/LocalDBConfig.de-AT.resx

```diff
@@ -0,0 +1,126 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
>  (Deutsch-Österreich)

I'm not sure why you change the naming style?

Reply:
Just applied the rename-change.